icevision/models/mmdet/utils.py

```diff
@@ -81,9 +81,9 @@ def param_groups(model):
         layers += [model.bbox_head]
 
         # YOLACT has mask_head and segm_head
-        if getattr(model, "mask_head"):
+        if getattr(model, "mask_head", False):
```

Comment:
why did we set this one to `False`?
same thing for line 86

Reply:
It's a bit confusing. I had a discussion with Dickson in his PR. I had the reaction as you.

Right now, there is a bug because False is missing

If False is missing it will add `mask_head` even for retinanet model for example.

The way to read is if `model` has a `mask_head` the statement is True otherwise set it to False.

Reply:
Sorry man.
I still don't get it.
> If False is missing it will add mask_head
But that's what we want.

Here how I understand the following statement:
```python
if getattr(model, "mask_head"): layers += [model.mask_head]
```
if the `model` object has a "mask_head" attribute then add `model.mask_head` to the layers list.

What am I missing?

Reply:
The False is the default value of `getattr` if the object does not have the attribute, otherwise, an `AttributeError` will be raise if no default is provided. We should change `getattr` for `hasattr` as it is the function intended for the usecase.

Reply:
Ah, alright! Got it. Thanks Frederik.